src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
My impression is that having everything under a subclass is that you would only inherit to your driver the pertinent classes and then chimera would test it with ``issubclass``... Although I thought this was the idea behind it, I never saw it implemented this way in chimera. 

I also remember convincing myself that this could live together with the "InstrumentFeatures", but I completely forgot the argument now.